src/generated/webhook-names.ts

```diff
@@ -41,8 +41,8 @@ export const webhookNames = [
   "installation.created",
   "installation.deleted",
   "installation.new_permissions_accepted",
-  "installation.suspend",
-  "installation.unsuspend",
+  "installation.suspended",
+  "installation.unsuspended",
```

Comment:
These values are incorrect, I fixed these in [octokit/webhooks#300](https://github.com/octokit/webhooks/pull/300)
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#installation

Reply:
Fixed in https://github.com/octokit/webhooks/pull/324